sections/Discussion.tex

```diff
@@ -0,0 +1,87 @@
+\section{Discussion}\label{sec:discussion}
+In this section, we investigate the different models to see how each meta-information field impacts the resulting topics.
+Firstly, we want to investigate the top words within each model.
+We have taken a random article from the dataset and visualized how the topics differ between the models. 
+Before investigating the article below, we define a specific color scheme for each model, seen in \autoref{tab:disc_color}.
+
+In the article in \autoref{fig:the_article}, we have highlighted the highest probable words within the three most occurring topics in the article.
+The article is about agriculture and how farmers are letting their cows out onto grass fields in September. 
+It also mentions a few different farms in the Northern part of Jutland and describes these in various ways.
+
+\begin{table}
+	\caption{Top 10 words for each model used in \autoref{tab:disc_color}.}
```

Comment:
caption needs to be updated,
I'm assuming its 'top 10 words, for top 3 topics' or perhaps 'top 10 words, for 3 random topics'?